views/random-beer.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <div class="beer">
+    
+    <div class="beer-img">
+      <img src="{{randomBeer.image_url}}" alt="">
+    </div>
+    <div class="beer-info">
+      {{#each responseFromAPI }}
+        {{> beerpartial }}
```

Comment:
Good use on partials! don't you have your img in your beerpartial already though?